src/components/td/td.jsx

```diff
@@ -77,8 +91,16 @@ Td.propTypes = {
   borderBottom: PropTypes.bool,
   borderLeft: PropTypes.bool,
   align: PropTypes.oneOf(['left', 'right', 'center']),
+  alignScreen: PropTypes.oneOf(['left', 'right', 'center']),
+  alignMobile: PropTypes.oneOf(['left', 'right', 'center']),
   /** By default a column will add ellipsis if the width is overflown. **Note:** this will only work if the child is a String. */
   ellipsis: PropTypes.bool,
+  tableDataTitle: PropTypes.string,
+  tableDataOrder: PropTypes.number,
```

Comment:
What do you think about renaming `tableDataOrder` to `order` (Or possibly `flexOrder`?) and `tableDataTitle` to `title`?